account-kit/rn-signer/src/client.ts

```diff
@@ -116,15 +148,61 @@ export class RNSignerClient extends BaseSignerClient<undefined> {
     this.eventEmitter.emit(params.connectedEventName, user, params.bundle);
     return user;
   }
-  override oauthWithRedirect(
+  override oauthWithRedirect = async (
     _args: Extract<OauthParams, { mode: "redirect" }>
```

Comment:
nit: `_args` implies the param isn't used. Since now it is, let's change it to `args`